widget/embedded/src/constants/quote.ts

```diff
@@ -0,0 +1,24 @@
+export const NETWORK_FEE = 'Network Fee';
+export const SWAPPER_FEE = 'Swapper Fee';
+export const AFFILIATE_FEE = 'Affiliate Fee';
+export const OUTBOUND_FEE = 'Outbound network fee';
+export const RANGO_FEE = 'Rango fee';
+
```

Comment:
Feel free to get rid of these constants, switch to a union type for stronger types and autocomplete, and then update the type wherever these constants were being used.
something like:
```ts
export type NonPayableFee =
  | 'Network Fee'
  | 'Swapper Fee'
  | 'Affiliate Fee'
  | 'Outbound network fee'
  | 'Rango fee';

export const NON_PAYABLE_FEES: {
  name: NonPayableFee;
  label: NonPayableFee | Omit<string, NonPayableFee>;
}[] = [
  {
    name: SWAPPER_FEE,
    label: 'Protocol Fee',
  },
  {
    name: AFFILIATE_FEE,
    label: AFFILIATE_FEE,
  },
  {
    name: OUTBOUND_FEE,
    label: 'Outbound Fee',
  },
  {
    name: RANGO_FEE,
    label: 'Rango Fee',
  },
];
```

Reply:
fixed